framework/src/utils/MooseMeshUtils.C

```diff
@@ -288,24 +288,30 @@ getExtraIDUniqueCombinationMap(const MeshBase & mesh,
   std::map<dof_id_type, dof_id_type> base_parsed_ids =
       MooseMeshUtils::getExtraIDUniqueCombinationMap(mesh, block_ids, extra_ids);
   // parsing extra ids based on ref_parsed_ids
-  std::set<std::pair<dof_id_type, dof_id_type>> unique_ids;
+  std::vector<std::pair<dof_id_type, dof_id_type>> unique_ids;
   std::map<std::pair<dof_id_type, dof_id_type>, std::set<const Elem *>> map_unique_id_to_elem;
```

Comment:
We possibly do not need to create this map because we can just loop the mesh again to retrieve the two ids and find the combined id. I am a little concerned of the insertion of this map of the size of this map is huge.

Reply:
While that is the case - this should resolve the immediate issue. I'm not sure if this is the best use of time unless someone else wants to pick it up.

Map insertion is O(log N) which is alright